2024-07-02 21:17:31,964 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2024-07-02 21:17:31,964 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /Users/renatomello/Projetos/fiap/warm-up/AI-instagram/codigo_alura/lib/python3.12/site-packages/instabot/bot/bot.py
2024-07-02 21:17:31,965 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2024-07-02 21:17:32,162 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2024-07-02 21:17:32,162 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2024-07-02 21:17:32,162 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2024-07-02 21:17:32,163 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
